nl/mail/chrome/messenger/localMsgs.properties

```diff
@@ -247,20 +247,20 @@ pop3PreviouslyEnteredPasswordIsInvalidPrompt=Geef een nieuw wachtwoord voor gebr
 
 ## @name POP3_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 ## @loc None
-4048=This POP3 server does not seem to support encrypted passwords. If you just set up the account, please try changing to 'Password, transmitted insecurely' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, this is a common scenario how someone could steal your password.
+4048=Het lijkt erop dat deze POP3 server geen versleutelde wachtwoorden ondersteunt. Probeer als u het account net heeft ingesteld om te wijzigen naar 'wachtwoord, onbeveiligd verzonden' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'. Als het eerst werkte en nu opeens niet meer, is dit een veelvoorkomend scenario hoe iemand uw wachtwoord kan stelen.
 
 ## @name POP3_AUTH_CHANGE_ENCRYPT_TO_PLAIN_SSL
 ## @loc None
-4049=This POP3 server does not seem to support encrypted passwords. If you just set up this account, please try changing to 'Normal password' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, please contact your email administrator or provider.
+4049=Het lijkt erop dat deze POP3 server geen versleutelde wachtwoorden ondersteunt. Probeer om als u dit account net heeft ingesteld om te wijzigen naar 'Normaal wachtwoord' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'. Als het eerst werkte en nu opeens niet meer, neem dan contact op met uw e-mail administrator of provider.
 
 ## @name POP3_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 ## @loc None
-4050=This POP3 server does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+4050=Deze POP3 server staat geen wachtwoorden in platte tekst toe. Probeer de wijzigen naar 'Versleuteld wachtwoord' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'.
```

Comment:
CORRECTION NEEDED, it should be: Deze POP3 server staat geen wachtwoorden in platte tekst toe. Probeer te wijzigen naar 'Versleuteld wachtwoord' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'.

- probeer te wijzigen vs. probeer de wijzigen